### PR TITLE
Add GPS Latitude and Longitude to EXIF data

### PR DESCRIPTION
**Pull request by abnersajr**
As requested in the issue #346 both `GPSLatitude` and `GPSLongitude` were added.
This information might be useful in scenarios what you want to display from where the picture was taken.

Both GraphQL and TypeScript types were added too.

### CODE REVIEW

File: src/set-fields-on-graphql-node-type.ts [hunk 1]
```diff
@@ -14,6 +14,21 @@ import fs from 'fs'
...
+  'ExposureTime',
+  'FNumber',
+  'FocalLength',
+  'GPSLatitude',
+  'GPSLongitude',
+  'ISO',
+  'LensModel',
+  'Model',
+  'ShutterSpeedValue',
+];
```
Comment (jessestuart) [inline src/set-fields-on-graphql-node-type.ts]: Codacy Issue found: [Extra semicolon. (semi)](https://app.codacy.com/app/jessestuart/gatsby-source-s3-image/pullRequest?prid=4611561)

**Comment by jessestuart** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/gatsby-source-s3-image/gatsby-source-s3-image/pullRequest?prid=4611561&bid=15470391)

**Comment by jessestuart** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/gatsby-source-s3-image/gatsby-source-s3-image/pullRequest?prid=4611561&bid=15470391)

File: src/set-fields-on-graphql-node-type.ts [hunk 2]
```diff
@@ -14,6 +14,21 @@ import fs from 'fs'
...
+  'ExposureTime',
+  'FNumber',
+  'FocalLength',
+  'GPSLatitude',
+  'GPSLongitude',
+  'ISO',
+  'LensModel',
+  'Model',
+  'ShutterSpeedValue'
+];
```
Comment (jessestuart) [inline src/set-fields-on-graphql-node-type.ts]: Codacy Issue found: [Extra semicolon. (semi)](https://app.codacy.com/app/jessestuart/gatsby-source-s3-image/pullRequest?prid=4611561)

**Comment by abnersajr** [comment]
Closing as it was never merged in more than 1 year. Feel free to re-open.

**Pull request closed without merging**